ext/curl/curl.stub.php

```diff
@@ -50,19 +50,19 @@ function curl_multi_exec(CurlMultiHandle $multi_handle, &$still_running): int {}
 
 function curl_multi_getcontent(CurlHandle $multi_handle): ?string {}
 
-/** @param int|null $msgs_in_queue */
-function curl_multi_info_read(CurlMultiHandle $multi_handle, &$msgs_in_queue = null): array|false {}
+/** @param int $queued_message_count */
+function curl_multi_info_read(CurlMultiHandle $multi_handle, &$queued_message_count = null): array|false {}
 
 function curl_multi_init(): CurlMultiHandle {}
 
 function curl_multi_remove_handle(CurlMultiHandle $multi_handle, CurlHandle $handle): int {}
 
 function curl_multi_select(CurlMultiHandle $multi_handle, float $timeout = 1.0): int {}
 
-function curl_multi_strerror(int $error_number): ?string {}
+function curl_multi_strerror(int $error_code): ?string {}
 
 #if LIBCURL_VERSION_NUM >= 0x071200 /* 7.18.0 */
-function curl_pause(CurlHandle $handle, int $bitmask): int {}
+function curl_pause(CurlHandle $handle, int $options): int {}
```

Comment:
```suggestion
function curl_pause(CurlHandle $handle, int $flags): int {}
```
Just as a general discussion point: Would it make sense to use `$options` for options arrays, and `$flags` for bitmasks?

Reply:
I like this idea!

Reply:
If we can be consistent across the language for that, it makes sense to me.  I'd have to double check and see if that's the case in other APIs.

Reply:
FYI: I 've just come across a few bitmask params with `$options` name in ext/dom.